e2e_playwright/st_vega_lite_chart_test.py

```diff
@@ -28,27 +28,27 @@ def test_vega_lite_chart(app: Page):
 
     for idx in range(VEGA_LITE_CHART_COUNT):
         chart = vega_lite_charts.nth(idx)
-        canvas = chart.locator("canvas").nth(0)
+        canvas = chart.locator("[role='graphics-document']").nth(0)
         expect(canvas).to_be_visible()
-        expect(canvas).to_have_class("marks")
+        expect(canvas.locator("svg")).to_have_class("marks")
```

Comment:
It seems like Vega Lite has changed from canvas to svg as the default renderer. Maybe rename the property to something other than canvas?